src/app/api/price-search/route.ts

```
@@ -0,0 +1,261 @@
+function toEUCJPEncoding(query: string) {
+  const eucJPEncoded = iconv.encode(query, "euc-jp");
+  return Array.from(eucJPEncoded)
+    .map((byte) => `%${byte.toString(16).toUpperCase().padStart(2, "0")}`)
+    .join("");
+}
+
+export async function GET(request: NextRequest) {
+  const params = {
+    query: request.nextUrl.searchParams.get("q"),
+    store: request.nextUrl.searchParams.get("store") || "dailyshot",
+    page: parseInt(request.nextUrl.searchParams.get("page") || "1"),
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)